docs/manual/guides/local-installation/ddev.de.md

```diff
@@ -78,6 +78,13 @@ Mit `ddev launch contao` kommst du direkt zur Administration.
 
 {{% /notice %}}
 
+
+{{% notice tip %}}
+Die [Contao Demo Webseite](https://demo.contao.org/) wird für die aktuell unterstützten Contao-Versionen gepflegt und kann optional 
+über Composer installiert werden. Hier findest du Informationen zur [Installation der Contao Demo](https://github.com/contao/contao-demo).
+{{% /notice %}}
```

Comment:
I think the previous one is the simple quickstart. The demo procedure is optional - therefore only as a later reference

Reply:
Yes, but you will need to execute that instead of `ddev composer create contao/managed-edition:5.3`. You cannot execute it after all the other described steps.